cobbler/actions/buildiso.py

```diff
@@ -217,10 +217,13 @@ def generate_netboot_iso(self, imagesdir, isolinuxdir, profiles=None, systems=No
                 else:
                     append_line += " autoyast=%s" % data["autoinstall"]
 
-            if dist.breed == "redhat":
+            if dist.breed in ["redhat", "redhatlegacy"]:
```

Comment:
Maybe get rid of separate `if`s and switch to `if ... elif ...`? Or maybe fix it in the separate PR.

Reply:
I would think refactoring work doesn't really belong to this patch, so I'd rather keep this for a separate PR.